pyston/utils/helpers.py

```diff
@@ -103,7 +110,27 @@ def serialized_data_to_python(data):
     elif isinstance(data, LAZY_SERIALIZERS):
         return serialized_data_to_python(data.serialize())
     elif isinstance(data, dict):
-        return OrderedDict(((key, serialized_data_to_python(val)) for key, val in data.items()))
+        return {key: serialized_data_to_python(val) for key, val in data.items()}
+    if isinstance(data, datetime.datetime):
```

Comment:
`DjangoJSONEncoder` is able to do this, why haven't you used it instead?

Reply:
because this is not serializaiton into JSON to to the python base types

Reply:
Really? I see you test it is a Python type and you convert it to string.

Reply:
I understand now, you want to use DjangoJSONEncoder only for this base types. This is clever. Yes I can do that. Unfortunatelly I already merged it. So I will upgrade it in the next version